tests/mbt/model/ccv.qnt

```diff
@@ -662,6 +848,161 @@ module ccv {
         }
     }
 
+    // Makes the given consumer slash the given validator for an infraction that happened while
+    // the provided vscId was in effect.
+    // Note that the model does not model the infractions themselves yet,
+    // e.g. we do not model validators actually double-signing or going down.
+    pure def sendSlashRequest(
+        currentState: ProtocolState,
+        consumer: Chain,
+        consumerNode: Node,
+        vscId: VscId,
+        downtime: bool): Result = {
+        val consumerState = currentState.consumerStates.get(consumer)
+        if (not(isRunningConsumer(consumer, currentState.providerState))) {
+            Err("Consumer is not currently a consumer - must have 'running' status!")
+        } else if (downtime and consumerState.outstandingDowntime.contains(consumerNode)) {
+            Err("Consumer already has outstanding downtime for validator!")
```

Comment:
In ICS, this condition is just perform an no-op. Are we simulating the same behaviour by returning an error here?

Reply:
An error means the model will not permit doing the action, and no trace will ever exhibit this behaviour.
I'll change that to be an actual noop - thanks for the diligence, great catch!